tests/acceptance/01_vars/02_functions/cache_name.cf

```diff
@@ -0,0 +1,46 @@
+#######################################################
+#
+# Test that the function result cache checks function name
+#
+#######################################################
+
+body common control
+{
+      inputs => { "../../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+#######################################################
+
+
+bundle agent init
+{
+}
+
+#######################################################
+
+bundle agent test
+{
+  meta:
+    "description" -> { "CFE-4244" }
+      string => "Test that the function result cache checks function name";
+
+  vars:
+    "res1" string => format("cfengine.com");
```

Comment:
I think these functions do not test the actual problem as they are not cached (only those with the `FNCALL_OPTION_CACHED` flag are IIRC).

Reply:
Ahaa, okey.